docs/resources/chain-integration.md

```diff
@@ -46,6 +46,266 @@
+| [`eth_getTransactionByBlockHashAndIndex`](#eth-gettransactionbyblockhashandindex) | Eth       | ✔           | ✔      |                    
+| [`eth_getTransactionReceipt`](#eth-gettransactionreceipt)                         | Eth       | ✔           | ✔      |                    
+| [`eth_newFilter`](#eth-newfilter)                                                 | Eth       | ✔           | ✔      |                    
+| [`eth_newBlockFilter`](#eth-newblockfilter)                                       | Eth       | ✔           | ✔      |                    
+| [`eth_newPendingTransactionFilter`](#eth-newpendingtransactionfilter)             | Eth       | ✔           | ✔      |                    
+| [`eth_uninstallFilter`](#eth-uninstallfilter)                                     | Eth       | ✔           | ✔      |                    
+| [`eth_getFilterChanges`](#eth-getfilterchanges)                                   | Eth       | ✔           | ✔      |                    
+| [`eth_getFilterLogs`](#eth-getfilterlogs)                                         | Eth       | ✔           | ✔      |                    
+| [`eth_getLogs`](#eth-getlogs)                                                     | Eth       | ✔           | ✔      |                    
+| `eth_getTransactionbyBlockNumberAndIndex`                                         | Eth       |             | ✔      |                    
+| `eth_submitHashrate`                                                              | Eth       |             |        |                    
+| `eth_getCompilers`                                                                | Eth       |             |        |                    
+| `eth_compileLLL`                                                                  | Eth       |             |        |                    
+| `eth_compileSolidity`                                                             | Eth       |             |        |                    
+| `eth_compileSerpent`                                                              | Eth       |             |        |                    
+| `eth_signTransaction`                                                             | Eth       |             |        |                    
+| [`eth_coinbase`](#eth-coinbase)                                                   | Eth       | ✔           |        |                    
+| [`eth_getProof`](#eth-getProof)                                                   | Eth       | ✔           |        |                    
+| [`eth_subscribe`](#eth-subscribe)                                                 | Websocket | ✔           |        |                    
+| [`eth_unsubscribe`](#eth-unsubscribe)                                             | Websocket | ✔           |        |                    
+
+:::tip
+Block Number can be entered as a Hex string, `"earliest"`, ``"latest"`` or `"pending"`.
```

Comment:
Can add an extra tip here on the endpoints.
Where the examples below are with a local node `http://localhost:8545`; user can also use our public full node for
`https://cronos-testnet-3.crypto.org:8545`.

Reply:
Added